NimbleExtension/Sources/JSONMapper/Models/JSON.swift

```diff
@@ -0,0 +1,95 @@
+//  JSON.swift
+//
+//  Created by Pirush Prechathavanich on 4/4/18.
+//  Copyright © 2018 Nimbl3. All rights reserved.
+//
+// swiftlint:disable multiline_function_chains
+
+enum JSON: Codable {
+
+    case string(String)
+    case int(Int)
+    case double(Double)
+    case bool(Bool)
+    case nested([String: JSON])
+    case array([JSON])
+    case `nil`
+
+    init(from decoder: Decoder) throws {
+        let container = try decoder.singleValueContainer()
+        self = try container.decodeIfPresent(String.self).map(JSON.string)
```

Comment:
Don't need to disable rule `swiftlint:disable multiline_function_chains`
Fix
```swift
self = try container.decodeIfPresent(String.self)
      .map(JSON.string) // enter new line here to fix swiftlint
      .or(...)
```

Reply:
having .map on the same line with so much chaining going on makes it look a lot more organised so I stand by my choice